src/views/Login.js

```diff
@@ -1,7 +1,110 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { withRouter, Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+import mutate from '../utils/mutate'
+import graphql from 'babel-plugin-relay/macro'
+
+import { Form, Container } from 'react-bootstrap'
+import { Text, Button, Layout } from '../styles'
+
+const mutation = graphql`
+  mutation LoginMutation($input: ObtainJSONWebTokenInput!){
+    tokenAuth(input: $input) {
+      token
+    }
+  } 
+`
+
+const Signup = ({ history, props }) => {
+  const [data, setData] = useState([])
+  const { username, password } = data
+  const [error, setError] = useState('none')
```

Comment:
Always use language level falsy values like `false` or `null` when declaring a falsy value. the string `'none'` will evaluate to `true` when evaluating `if (error)`. Using `null` feels the most appropriate in this case as it represents the lack of any value.

Reply:
This is still a problem